modules/storage-persistent-storage-pv.adoc

```diff
@@ -0,0 +1,293 @@
+the pods that use these volumes are deleted.
+====
+
+.Supported access modes for PVs
+[cols=",^v,^v,^v", width="100%",options="header"]
+|===
+|Volume Plug-in  |ReadWriteOnce  |ReadOnlyMany  |ReadWriteMany
+|AWS EBS  | ✅ | - |  -
+|Azure File | ✅ | ✅ | ✅
+|Azure Disk | ✅ | - | -
+|Ceph RBD  | ✅ | ✅ |  -
+|Fibre Channel  | ✅ | ✅ |  -
+|GCE Persistent Disk  | ✅ | - |  -
+|GlusterFS  | ✅ | ✅ | ✅
+|HostPath  | ✅ | - |  -
+|iSCSI  | ✅ | ✅ |  -
+|NFS  | ✅ | ✅ | ✅
+|Openstack Cinder  | ✅ | - |  -
+|VMWare vSphere | ✅ | - |  -
+|Local | ✅ | - |  -
+|===
+
+[NOTE]
+====
+Use a recreate deployment strategy for pods that rely on AWS EBS.
+// GCE Persistent Disks, or Openstack Cinder PVs. 
+====
+
+
+ifdef::openshift-dedicated,openshift-online[]
+[[pv-restrictions]]
+== Restrictions
+
+The following restrictions apply when using PVs with {product-title}:
+endif::[]
+
+ifdef::openshift-dedicated[]
+[IMPORTANT]
+====
+ * PVs are provisioned with either EBS volumes (AWS) or GCP storage (GCP), 
+depen ding on where the cluster is provisioned.
+ * Only RWO access mode is applicable, as EBS volumes and GCE Persistent 
+Disks cannot be mounted to multiple nodes.
+ * *emptyDir* has the same lifecycle as the pod:
+   ** *emptyDir* volumes survive container crashes/restarts.
+   ** *emptyDir* volumes are deleted when the pod is deleted.
+====
+endif::[]
+
+ifdef::openshift-online[]
+[IMPORTANT]
+====
+ * PVs are provisioned with EBS volumes (AWS).
+ * Only RWO access mode is applicable, as EBS volumes and GCE Persistent 
+Disks cannot be mounted to multiple nodes.
+ * Docker volumes are disabled.
+   ** VOLUME directive without a mapped external volume fails to be 
+instantiated
+.  
+ * *emptyDir* is restricted to 512 Mi per project (group) per node.
+   ** A single pod for a project on a particular node can use up to 512 Mi 
+of *emptyDir* storage.
+   ** Multiple pods for a project on a particular node share the 512 Mi of 
+*emptyDir* storage. 
+ *  *emptyDir* has the same lifecycle as the pod:
+   ** *emptyDir* volumes survive container crashes/restarts.
+   ** *emptyDir* volumes are deleted when the pod is deleted.
+====
+endif::[]
+
+[[pv-reclaim-policy]]
+== Reclaim policy
+
+The following table lists current reclaim policies:
+
+.Current reclaim policies
+[cols="1,2",options="header"]
+|===
+
+|Reclaim policy 
+|Description
+
+|Retain
+|Manual reclamation
+
+|===
+
+[WARNING]
+====
+If you do not want to retain all pods, use dynamic provisioning.
+====
+
+[[pv-phase]]
+== Phase
+
+Volumes can be found in one of the following phases:
+
+.Volume phases
+[cols="1,2",options="header"]
+|===
+
+|Phase 
+|Description
+
+|Available
+|A free resource not yet bound to a claim.
+
+|Bound
+|The volume is bound to a claim.
+
+|Released
+|The claim was deleted, but the resource is not yet reclaimed by the
+cluster.
+
+|Failed
+|The volume has failed its automatic reclamation.
+
+|===
+
+The CLI shows the name of the PVC bound to the PV.
```

Comment:
This sentence seems weird to me. Should it say something like "You can see the name of the PVC bound to the PV using the following CLI command:" and then list the CLI command?

Reply:
I don't have an active cluster right now. I've filed https://jira.coreos.com/browse/OSDOCS-332 to follow up on this once one is available. Marking resolved for now, and will revisit this issue under the above JIRA.